ganttproject/src/biz/ganttproject/impex/csv/GanttCSVExport.java

```diff
@@ -62,13 +60,30 @@
  * @author athomas
  */
 public class GanttCSVExport {
+
+  public enum FileExtensionEnum {
+    CSV() {
+      @Override
+      public String toString() {
```

Comment:
I recommend to add a member field, constructor argument and implement toString() like `return myExtension`, without anonymous classes.